header/p2p/helpers.go

```diff
@@ -0,0 +1,91 @@
+package p2p
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/protocol"
+
+	"github.com/celestiaorg/go-libp2p-messenger/serde"
+
+	"github.com/celestiaorg/celestia-node/header"
+	p2p_pb "github.com/celestiaorg/celestia-node/header/p2p/pb"
+)
+
+func protocolID(protocolSuffix string) protocol.ID {
+	return protocol.ID(fmt.Sprintf("/header-ex/v0.0.3/%s", protocolSuffix))
+}
+
+// sendMessage opens the stream to the given peers and sends ExtendedHeaderRequest to fetch ExtendedHeaders.
+// As a result sendMessage returns ExtendedHeaderResponse, the size of fetched data,
+// the duration of the request and an error.
+func sendMessage(
```

Comment:
The code in here is still not grouped, but that's nit

Reply:
done